packages/atomic/src/components/insight/atomic-insight-interface/atomic-insight-interface.tsx

```diff
@@ -23,6 +17,12 @@ import {
 } from '../../common/interface/interface-common';
 import {AtomicInsightStore, createAtomicInsightStore} from './store';
 import {getAnalyticsConfig} from './analytics-config';
+import {
+  InsightLogLevel,
+  InsightEngine,
+  InsightEngineConfiguration,
+  buildInsightEngine,
+} from '..';
```

Comment:
🤔 why these paths changed?

Reply:
so, if I remember correctly, right before I went on holiday we discussed the problems with importing/exporting vars with the same name in the auto-generated components file. If you look at the file where we are importing it from, you will see that I'm renaming everything that has a generic name (aka doesn't include Insight)
```js
export {
  AnalyticsConfiguration as InsightAnalyticsConfiguration,
  LogLevel as InsightLogLevel,
  InsightEngine,
  InsightEngineConfiguration,
  buildInsightEngine,
  InsightResultList,
  InsightResultListState,
  buildInsightResultList,
  Result as InsightResult,
  ResultTemplate as InsightResultTemplate,
  ResultTemplateCondition as InsightResultTemplateCondition,
  ResultTemplatesManager as InsightResultTemplatesManager,
  buildResultTemplatesManager as insightBuildResultTemplatesManager,
  buildInsightSearchBox,
  InsightSearchBox,
  SearchBoxState as InsightSearchBoxState,
} from '@coveo/headless/insight';
```